handlers_contract_creator.go

```diff
@@ -36,7 +37,7 @@ func (c *Client) GetContractCreator(ctx context.Context, addr common.Address) (*
 		},
 		FieldSelection: types.FieldSelection{
 			Block:       []string{"number"},
-			Transaction: []string{"hash"},
+			Transaction: []string{"hash", "from"},
```

Comment:
Ideally include only needed fields to the query, but this is definitely a good fix 👍

Reply:
In general these are the fields necessary for contract creation discovery. I've went and did it pretty much on the same way ots_getContractCreator is doing on actual nodes.